core/templates/dev/head/editor/feedback_tab/feedback_tab.html

```diff
@@ -1,8 +1,8 @@
 <md-card class="oppia-editor-card" ng-controller="FeedbackTab">
   <div ng-if="!activeThread">
-    <div ng-if="(threadData.threadList | filter:!has_suggestion).length > 0">
+    <div ng-if="(threadData.threadList | filter:{suggestion_id:'!undefined'}).length > 0">
```

Comment:
Using `suggestion_id` does not appear to work. Added a comment explaining the filter syntax.